### PR TITLE
Add ignore-name-format flag to interleave-reads.py

### PR DESCRIPTION
**Pull request by camillescott**
Addresses #1294 by adding a flag to turn off renaming.

### CODE REVIEW

**Comment by camillescott** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Do the changes respect streaming IO? (Are they
  tested for streaming IO?)
- [ ] Is the Copyright year up to date?

**Comment by camillescott** [comment]
@ctb @luizirber ready for review

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+2015-09-17  Camille Scott  <camille.scott.w@gmail.com>
+   * scripts/interleave-reads.py: Added --no-reformat flah
```
Comment (luizirber) [inline ChangeLog]: flah -> flag

**Comment by luizirber** [comment]
LGTM

**Comment by camillescott** [comment]
Thanks!

**Pull request merged**